dev-db/percona-xtrabackup/percona-xtrabackup-2.4.5.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-${PV}/s
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="amd64 x86"
```

Comment:
you don't need a pull request for this, the members of the corresponding arch teams will test the package and stabilize it for you